cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -2645,31 +2645,31 @@ pPoolInfluence :: Parser Rational
 pPoolInfluence =
     Opt.option readRational
       (  Opt.long "pool-influence"
-      <> Opt.metavar "DOUBLE"
+      <> Opt.metavar "RATIONAL"
       <> Opt.help "Pool influence."
       )
 
 pTreasuryExpansion :: Parser Rational
 pTreasuryExpansion =
     Opt.option readRationalUnitInterval
       (  Opt.long "treasury-expansion"
-      <> Opt.metavar "DOUBLE"
+      <> Opt.metavar "RATIONAL"
       <> Opt.help "Treasury expansion."
       )
 
 pMonetaryExpansion :: Parser Rational
 pMonetaryExpansion =
     Opt.option readRationalUnitInterval
       (  Opt.long "monetary-expansion"
-      <> Opt.metavar "DOUBLE"
+      <> Opt.metavar "RATIONAL"
       <> Opt.help "Monetary expansion."
       )
 
 pDecentralParam :: Parser Rational
 pDecentralParam =
     Opt.option readRationalUnitInterval
       (  Opt.long "decentralization-parameter"
-      <> Opt.metavar "DOUBLE"
```

Comment:
Let's leave it as `DOUBLE` as we can assume some technical knowledge if somebody is using `cardano-cli`.

Reply:
Maybe DECIMAL?

Reply:
The reason is we accept strings like "31/57". I can't call it DOUBLE[-PRECISION FLOATING POINT].